lib/Doctrine/DBAL/Migrations/Version.php

```diff
@@ -407,7 +407,8 @@ private function executeRegisteredSql($dryRun = false, $timeAllQueries = false)
                         $this->outputWriter->write('     <comment>-></comment> ' . $query);
                         $this->connection->executeQuery($query);
                     } else {
-                        $this->outputWriter->write(sprintf('    <comment>-</comment> %s (with parameters)', $query));
+                        $parameters = implode(', ', $this->params[$key]);
+                        $this->outputWriter->write(sprintf('    <comment>-</comment> %s (%s)', $query, $parameters));
```

Comment:
This whole bit (including the `if/else` check for the params) could be replaced with [`$this->outputSqlQuery($key, $query)`](https://github.com/mikeSimonson/migrations/blob/1f9e1ed1af23565ac0028c9c379bc8342db96e3e/lib/Doctrine/DBAL/Migrations/Version.php#L437-L449).

I added that in #477 -- uses the doctrine types system to convert the params into something printable.